CriteoPublisherSdk/Tests/UnitTests/Network/CR_BidRequestSerializerSwiftTests.swift

```diff
@@ -136,12 +138,27 @@ class CR_BidRequestSerializerSwiftTests: XCTestCase {
     XCTAssertEqual(user[NSString.mopubConsent]!, "Privacy consent for mopub")
   }
 
+  func testBodyWithContext() {
+    context = CRContextData(dictionary:[
+      "a.a": "foo",
+      "b": "bar",
+    ])
+
+    let expected = [
+      "a": [
+        "a": "foo",
+      ],
+      "b": "bar",
+    ] as [String: AnyHashable]
+
+    let body = generateBody()
+
+    let publisher = body["publisher"]! as! [String: AnyHashable]
+    XCTAssertEqual(publisher["ext"], expected)
+  }
+
   private func generateBody() -> [String: AnyHashable] {
-    return serializer.body(
-      with: request,
-      consent: consent,
-      config: config,
-      deviceInfo: deviceInfo) as! [String: AnyHashable]
+    return serializer.body(with: request, consent: consent, config: config, deviceInfo: deviceInfo, context: context) as! [String: AnyHashable]
```

Comment:
Is this the result of `bundle exec fastlane format`? If not, could you please follow the repository formatting?

Reply:
My swift-format has error unfortunately :/ It's weird because I've just update it via brew.
Let me reformat this.

Reply:
Ah I guess swift-format is still not updated for Xcode 12, probably a reason why I guess I didn't enforce it on CI.
Forget it then, I'll reformat the repo at some point